tests/README.md

```diff
@@ -0,0 +1,33 @@
+# Robot Framework Automated Hardware Tests
+
+These are tests that are run with Robot Framework and involve a shell
+interactions and usually using reference hardware.
+
+## Glossary
+
+DUT - Device Under Test
+PHiLIP - The reference hardware to test against
+
+## make overrides
+
+Variable     | Description                                     | Example(s)
+-------------|-------------------------------------------------|------------------------------------
+BOARD        | The name of the DUT board _(required)_          | nucleo-f030r8, samr21-xpro
+PORT         | The serial port of the DUT                      | /dev/ttyACM0, /dev/ttyUSB1
+BAUD         | The baudrate of the DUT serial connection       | 115200, 9600
+PHILIP_PORT  | The serial port of PHiLIP _(required)_          | /dev/ttyACM0
+CMD_TIMEOUT  | The time (s) for a command to respond           | 0.5, 1
+RESET_WAIT   | The time (s) to wait after a reset              | 0, 3
+CONNECT_WAIT | The time (s) to wait after connecting to serial | 0, 5
+RESET        | The command used to reset the device            | 'python3 -m philip_pal --dut_reset'
+RESET_FLAGS  | Flags for the reset command                     | /dev/ttyACM0, ${PHILIP_PORT}
+
+The `RESET_WAIT` is useful for speeding up tests if a longer reset wait period
+is not needed like in the case of using the PHiLIP reset. Then `RESET_WAIT`
+can be 0. For some boards, such as the arduino, the bootloader is activated
```

Comment:
`the Arduino(s)`, start with capital as it is a name or trademark ?

Reply:
yup